DPG/Tasks/qaK0sTrackingEfficiency.cxx

```diff
@@ -70,36 +91,39 @@ struct qaK0sTrackingEfficiency {
     registry.fill(HIST("h_EventCounter"), 1.);
 
     for (auto& v0 : fullV0s) {
-      if (v0.v0cosPA(collision.posX(), collision.posY(), collision.posZ()) > v0cospa) {
-
-        auto reconegtrack = v0.negTrack_as<PIDTracks>();
-        auto recopostrack = v0.posTrack_as<PIDTracks>();
-
-        if (TMath::Abs(v0.yK0Short()) < rapidity) {
-          if (reconegtrack.tpcNSigmaPi() < nSigTPC && recopostrack.tpcNSigmaPi() < nSigTPC) {
-            registry.fill(HIST("Test/h_R"), v0.v0radius());
-            registry.fill(HIST("Test/h_pT"), v0.pt());
-            registry.fill(HIST("Test/h_mass"), v0.mK0Short());
-
-            int negIBNhits = 0, posIBNhits = 0;
-            for (unsigned int i = 0; i < 3; i++) {
-              if (reconegtrack.itsClusterMap() & (1 << i)) {
-                negIBNhits++;
-              }
-              if (recopostrack.itsClusterMap() & (1 << i)) {
-                posIBNhits++;
-              }
-            }
-            bool negHasIB = (bool)negIBNhits;
-            bool posHasIB = (bool)posIBNhits;
-            registry.fill(HIST("Test/h_negStatus"), negHasIB);
-            registry.fill(HIST("Test/h_posStatus"), posHasIB);
-            registry.fill(HIST("Test/h_negIBhits"), negIBNhits);
-            registry.fill(HIST("Test/h_posIBhits"), posIBNhits);
-
-            registry.fill(HIST("h5_RpTmassStatus"), v0.v0radius(), v0.pt(), v0.mK0Short(), negHasIB, posHasIB);
+
+      auto reconegtrack = v0.negTrack_as<PIDTracks>();
+      auto recopostrack = v0.posTrack_as<PIDTracks>();
+
+      if (acceptV0(v0, reconegtrack, recopostrack, collision)) {
+        registry.fill(HIST("Test/h_R"), v0.v0radius());
+        registry.fill(HIST("Test/h_pT"), v0.pt());
+        registry.fill(HIST("Test/h_mass"), v0.mK0Short());
+
+        bool negHasITS = reconegtrack.hasITS();
+        bool posHasITS = reconegtrack.hasITS();
+        registry.fill(HIST("Test/h_negITSStatus"), negHasITS);
+        registry.fill(HIST("Test/h_posITSStatus"), posHasITS);
```

Comment:
Very minor comment, up to you if you want to change: if you do not use `negHasITS` and `posHasITS` anymore in the code, you can directly use `reconegtrack.hasITS()` and `recopostrack.hasITS()` in the histogram filling.